pep-0655.rst

```diff
@@ -14,16 +14,21 @@ Post-History: 31-Jan-2021, 11-Feb-2021, 20-Feb-2021, 26-Feb-2021, 17-Jan-2022, 2
 Abstract
 ========
 
-:pep:`589` defines syntax
-for declaring a TypedDict with all required keys and syntax for defining
+:pep:`589` defines notation
+for declaring a TypedDict with all required keys and notation for defining
 a TypedDict with :pep:`all potentially-missing keys <589#totality>` however it
-does not provide any syntax to declare some keys as required and others
-as potentially-missing. This PEP introduces two new syntaxes:
+does not provide any notation to declare some keys as required and others
+as potentially-missing. This PEP introduces two new notations:
 ``Required[]`` which can be used on individual items of a
 TypedDict to mark them as required, and
 ``NotRequired[]`` which can be used on individual items
 to mark them as potentially-missing.
 
+No Python grammar changes are made by this PEP. Correct usage
+of required and potentially-missing keys of TypedDicts is intended to be
+enforced only by static type checkers and in particular is not enforced by
```

Comment:
```suggestion
enforced only by static type checkers, and is not required by
```

Needs a comma, eliminate redundant/verbose wording, and avoid repetition while setting off the two statements more clearly (with the need for "in particular").

Reply:
Did apply an alternate rephrasing that avoids the use of "required", which has a special jargon meaning for much of this PEP.